samples/firefox-accounts/build.gradle

```diff
@@ -51,4 +51,5 @@ dependencies {
 
     implementation "com.android.support:appcompat-v7:${rootProject.ext.dependencies['supportLibraries']}"
     implementation "com.android.support:customtabs:${rootProject.ext.dependencies['supportLibraries']}"
+    implementation 'com.android.support:support-v4:27.1.1'
```

Comment:
We managed all versions the root project. So, this needs to be replaces with:
`implementation "com.android.support:support-v4:${rootProject.ext.dependencies['supportLibraries']}"`